pkg/version/version.go

```diff
@@ -0,0 +1,103 @@
+package version
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/aquasecurity/trivy-db/pkg/metadata"
+	javadb "github.com/aquasecurity/trivy-java-db/pkg/db"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/policy"
+)
+
+var (
+	Version = "dev"
+)
+
+type VersionInfo struct {
+	Version         string             `json:",omitempty"`
+	VulnerabilityDB *metadata.Metadata `json:",omitempty"`
+	JavaDB          *metadata.Metadata `json:",omitempty"`
+	PolicyBundle    *policy.Metadata   `json:",omitempty"`
+}
+
+func formatDbMetadata(title string, meta metadata.Metadata) string {
+	return fmt.Sprintf(`%s:
+  Version: %d
+  UpdatedAt: %s
+  NextUpdate: %s
+  DownloadedAt: %s
+`, title, meta.Version, meta.UpdatedAt.UTC(), meta.NextUpdate.UTC(), meta.DownloadedAt.UTC())
+}
+
+func (v *VersionInfo) String() string {
+	output := fmt.Sprintf("Version: %s\n", v.Version)
+	if v.VulnerabilityDB != nil {
+		output += formatDbMetadata("Vulnerability DB", *v.VulnerabilityDB)
+	}
+	if v.JavaDB != nil {
+		output += formatDbMetadata("Java DB", *v.JavaDB)
+	}
+	if v.PolicyBundle != nil {
+		output += v.PolicyBundle.String()
+	}
+	return output
+}
+
+func BuildVersionInfo(appVersion string, cacheDir string) VersionInfo {
```

Comment:
nit:

```suggestion
func NewVersionInfo(appVersion string, cacheDir string) VersionInfo {
```

Reply:
Also, do we need to pass `appVersion` here? There is a version info in this package?